src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a class="logo__link" href="home">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+            class="logo"
+          >
+        </a>
+
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="phones">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="pc"
+              data-qa="hover"
+            >Ноутбуки и компьютеры
+            </a>
```

Comment:
It's looks better in such way, isn't it?
```suggestion
            <a
              class="nav__link"
              href="pc"
              data-qa="hover"
            >
            	Ноутбуки и компьютеры
            </a>
```